src/app.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+const fs = require('fs');
+
+function copyFiles() {
+  const [fileSource, fileDestination] = process.argv.slice(2);
+
+  if (!fileDestination) {
+    // eslint-disable-next-line no-console
+    console.error('Provide both file source path and file destination path');
+
+    return;
+  }
+
+  if (fileSource === fileDestination) {
+    return;
+  }
+
+  fs.cp(fileSource, fileDestination, (error) => {
```

Comment:
The `fs.cp` method does not exist in the `fs` module. You should use `fs.copyFile` instead for copying files. Update this line to use `fs.copyFile(fileSource, fileDestination, (error) => {...})`.